server/pulp/server/async/app.py

```diff
@@ -136,7 +136,12 @@ def get_resource_manager_lock(name):
             lock.save()
 
             msg = _("Resource manager '%s' has acquired the resource manager lock") % name
-            _logger.info(msg)
+            _logger.debug(msg)
```

Comment:
Nah it won't.  Notice the ```lock.save()``` above it - if it succeeds it will never happen again because of the ```break```.  If it fails, it skips to the exception handler.